src/app/statuses.rs

```diff
@@ -63,3 +70,32 @@ pub async fn reload(_device: AdminDevice, mut db: Connection<Db>) -> Result<Tama
 	Status::ping_servers_and_save(&mut db).await;
 	Ok(TamanuHeaders::new(()))
 }
+
+#[post("/status/<server_id>/<current_version>")]
+pub async fn create(
+	_device: ServerDevice,
+	remote_addr: SocketAddr,
```

Comment:
hmm, no, i'd like IpAddr specifically because SocketAddr behind a load balancer will get the ip of the load balancer https://api.rocket.rs/v0.5/rocket/request/struct.Request#method.remote